e2e/go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/cosmos/cosmos-sdk v0.47.0
 	github.com/cosmos/gogoproto v1.4.6
 	github.com/cosmos/ibc-go/v7 v7.0.0
-	github.com/cosmos/interchain-accounts v0.5.0
```

Comment:
there was an error I'm not quite sure I can explain happening suddenly

```
On branch cian/issue#3445-add-verifiable-upgrade-type-and-validate-basic-functions
        downloaded: h1:b9P3j2ZPziqYUYzb7zHVucZg84EfiGY+T+fPgszVxfI=                                                         [135/1987]
        sum.golang.org: h1:0Pzv5xHq+TVY7ExRIsrzJ33+t22TUoJ1k8UgSxvv1X4=

SECURITY ERROR
This download does NOT match the one reported by the checksum server.
The bits may have been replaced on the origin server, or an attacker may
have intercepted the download attempt.

For more information, see 'go help module-auth'.

github.com/cosmos/ibc-go/e2e/testsuite imports
        github.com/strangelove-ventures/interchaintest/v7/chain/cosmos imports
        github.com/cosmos/cosmos-sdk/x/upgrade imports
        github.com/cosmos/cosmos-sdk/x/upgrade/client/cli imports
        github.com/cosmos/cosmos-sdk/x/upgrade/plan imports
        github.com/hashicorp/go-getter imports
        cloud.google.com/go/storage tested by
        cloud.google.com/go/storage.test imports
        cloud.google.com/go/httpreplay imports
        cloud.google.com/go/httpreplay/internal/proxy imports
        github.com/google/martian/v3/mitm: github.com/cosmos/interchain-accounts@v0.5.0: verifying go.mod: checksum mismatch
        downloaded: h1:b9P3j2ZPziqYUYzb7zHVucZg84EfiGY+T+fPgszVxfI=
        sum.golang.org: h1:0Pzv5xHq+TVY7ExRIsrzJ33+t22TUoJ1k8UgSxvv1X4=

SECURITY ERROR
This download does NOT match the one reported by the checksum server.
The bits may have been replaced on the origin server, or an attacker may
have intercepted the download attempt.

For more information, see 'go help module-auth'.

github.com/cosmos/ibc-go/e2e/testsuite imports
        github.com/strangelove-ventures/interchaintest/v7/chain/cosmos imports
        github.com/cosmos/cosmos-sdk/x/upgrade imports
        github.com/cosmos/cosmos-sdk/x/upgrade/client/cli imports
        github.com/cosmos/cosmos-sdk/x/upgrade/plan imports
        github.com/hashicorp/go-getter imports
        github.com/aws/aws-sdk-go/service/s3 imports
        github.com/aws/aws-sdk-go/aws/awsutil imports
        github.com/jmespath/go-jmespath tested by
        github.com/jmespath/go-jmespath.test imports
        github.com/jmespath/go-jmespath/internal/testify/assert: github.com/cosmos/interchain-accounts@v0.5.0: verifying go.mod: check
sum mismatch
        downloaded: h1:b9P3j2ZPziqYUYzb7zHVucZg84EfiGY+T+fPgszVxfI=
        sum.golang.org: h1:0Pzv5xHq+TVY7ExRIsrzJ33+t22TUoJ1k8UgSxvv1X4=

SECURITY ERROR
This download does NOT match the one reported by the checksum server.
The bits may have been replaced on the origin server, or an attacker may
have intercepted the download attempt.

For more information, see 'go help module-auth'.

```

It looks like some sort of mismatch in v0.5.0 in the one we've imported and the one that is imported in interchaintest.